.github/workflows/rust.yml

```diff
@@ -0,0 +1,48 @@
+name: Rust test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        rust: [stable, nightly, beta]
+
+    steps:
+      - uses: hecrj/setup-rust-action@v1
+        with:
+          rust-version: ${{ matrix.rust }}
+      - uses: actions/checkout@master
+      - name: Build
+        run: cargo build --examples --lib --bins
+      - name: Test
+        run: cargo test --verbose --examples --lib --bins
+  clippy:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    steps:
+      - uses: actions/checkout@master
+      - name: Install clippy
+        run: rustup component add clippy
+      - name: Run clippy
+        run: cargo clippy --all-targets -- --deny clippy::all
```

Comment:
clippy should run on rust stable branch only

Reply:
It does now

Reply:
got it 👍